suite-native/app/Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby '>= 3.0'
+ruby ">= 2.6.10"
```

Comment:
Just curious: is it necessary to downgrade?

Reply:
It's copypasted from default RN template for 0.73 but it's probablny not necessary.

Reply:
fixed